src/util.cpp

```diff
@@ -1562,7 +1562,7 @@ QString GetVersionAndNameStr ( const bool bDisplayInGui )
         strVersionText += " *** ";
     }
 
-    strVersionText += APP_NAME + QCoreApplication::tr ( ", Version " ) + VERSION;
+    strVersionText += QCoreApplication::tr ( "%1, Version %2" ).arg ( APP_NAME ).arg ( VERSION );
```

Comment:
I suggest using the 3rd `disambiguation` argument here to ensure translators know what `%1` and `%2` stands for. Something like:

```suggestion
    strVersionText += QCoreApplication::tr ( "%1, Version %2", "%1 is app name, %2 is version number" ).arg ( APP_NAME ).arg ( VERSION );
```

Reply:
Agree. Thanks